src/components/Table.js

```diff
@@ -44,14 +44,7 @@ function Table({ tableData, formatter }) {
     return (
         <>
             <table className="table px-5">
```

Comment:
spread the rest props 
```jsx
            <table {...rest}>
````
remove `className` from here and add it to `Table` components in ListCustomer, ListItems and ListInvoices.